modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -981,8 +981,8 @@ private PartitionReplicaListener createReplicaListener(
                 schemaSyncService,
                 catalogService,
                 new DirectCatalogTables(catalogService),
-                placementDriver
-        );
+                placementDriver,
+                clusterNodeResolver);
```

Comment:
We should get rid of clusterNodeResolver wherever possible. Messaging service has overloaded versions of invoke methods that use consistentNodeId instead of clusterNode as a recipient parameter. We should favour using overloaded versions and thus if it's possible skip clusterNodeResolver.